setup.py

```diff
@@ -88,6 +88,16 @@ def get_extensions():
 
     return ext_modules
 
+# Mimic code from torchvision https://github.com/pytorch/vision/blob/143d078b28f00471156a4e562dd3836370acc9ee/setup.py#L58
+pytorch_dep = "torch"
+if os.getenv("PYTORCH_VERSION"):
```

Comment:
Lgtm, would need to check in workflow that PYTORCH_VERSION is actually being set correcty when running nightly build